en/middleware/micrortps.md

```diff
@@ -1,384 +1,3 @@
-# RTPS/DDS Interface: PX4-Fast RTPS(DDS) Bridge
+<Redirect to="microdds" />
```

Comment:
Note, deleted content. Not relevant once PR goes in. The note on the microDDS page explains what happened to it and where you go now if you're in old versions